README.md

```diff
@@ -18,40 +18,74 @@ Some of these are tricky!  Don't trust your first instinct.
 
```

Comment:
Good work on writing out your explanations. You had a lot of the results correct, but your reasonings for some of them were a bit off target. Take a look at the homework solutions and the comments below for more specific reasons. Also, try tackling these equations in smaller steps in the console so you have an idea of what is happening at each operation.

Additionally, in future homework assignments remember that, after cloning your git repository, you'll want to make your own folder to store your homework answers in.